docs/updating/from_4.0/to_4.1.md

```diff
@@ -5,89 +5,161 @@ latest_tag: '4.1.5'
     ```
 
 === "[[= product_name_exp =]]"
 
     ``` bash
-    composer require ibexa/experience:[[= latest_tag =]] --with-all-dependencies --no-scripts
+    composer require ibexa/experience:4.0.8 --with-all-dependencies --no-scripts
+    composer recipes:install ibexa/experience --force -v
     ```
 
 === "[[= product_name_com =]]"
 
     ``` bash
-    composer require ibexa/commerce:[[= latest_tag =]] --with-all-dependencies --no-scripts
+    composer require ibexa/commerce:4.0.8 --with-all-dependencies --no-scripts
+    composer recipes:install ibexa/commerce --force -v
     ```
 
-### Update Flex server
-
-The `flex.ibexa.co` Flex server has been disabled. 
-If you are using earlier v4.x versions, and you have not done it before, 
-you have to update your Flex server.
-
-To do it, in your `composer.json` check whether the `https://flex.ibexa.co` endpoint is still listed in `extra.symfony.endpoint`. 
-If so, replace it with the new [`https://api.github.com/repos/ibexa/recipes/contents/index.json?ref=flex/main`](https://github.com/ibexa/website-skeleton/blob/v4.1.5/composer.json#L96) endpoint.
-
-You can do it manually, or by running the following command:
+Next, run:
 
 ``` bash
-composer config extra.symfony.endpoint "https://api.github.com/repos/ibexa/recipes/contents/index.json?ref=flex/main"
+composer run post-install-cmd
+mysql -u <username> -p <password> <database_name> < vendor/ibexa/installer/upgrade/db/mysql/ibexa-4.0.3-to-4.0.4.sql
 ```
-    
-### Continue with updating the app
 
-Run the following command:
+## Update the app to v4.1.0
+
+When you have the v4.0 version, you can update to v4.1.0:
 
 === "[[= product_name_content =]]"
 
     ``` bash
+    composer require ibexa/content:4.1.0 --with-all-dependencies --no-scripts
     composer recipes:install ibexa/content --force -v
     ```
 
 === "[[= product_name_exp =]]"
 
     ``` bash
+    composer require ibexa/experience:4.1.0 --with-all-dependencies --no-scripts
     composer recipes:install ibexa/experience --force -v
     ```
 
 === "[[= product_name_com =]]"
 
     ``` bash
+    composer require ibexa/commerce:4.1.0 --with-all-dependencies --no-scripts
     composer recipes:install ibexa/commerce --force -v
     ```
 
 The `recipes:install` command installs new YAML configuration files. Look through the old YAML files and move your custom configuration to the relevant new files.
 
-## Update the database
+Next, run:
+
+``` bash
+composer run post-install-cmd
+```
+
+
+### Update the database
 
 [[% include 'snippets/update/db/db_backup_warning.md' %]]
 
 Apply the following database update scripts:
 
-### Ibexa DXP
+=== "MySQL"
+    ``` bash
+    mysql -u <username> -p <password> <database_name> < vendor/ibexa/installer/upgrade/db/mysql/ibexa-4.0.0-to-4.1.0.sql
+    ```
+
+=== "PostgreSQL"
+
+    ``` bash
+    psql <database_name> < vendor/ibexa/installer/upgrade/db/postgresql/ibexa-4.0.0-to-4.1.0.sql
+    ```
+
+## Update the app to latest version of v4.1
```

Comment:
IMVHO this procedure does not fit here as it describes 4.1.x to latest and has been described in this article:
https://doc.ibexa.co/en/latest/updating/from_4.1/update_from_4.1/
Why not update the above procedure with any differences compared to the procedure that have documented here and just link from here to this other article?

Reply:
In https://github.com/ezsystems/developer-documentation/pull/1701/files#diff-ceaa305879e55a9235983a5c64497f5748fe12470495e454dbfd64db9cbba9b2R7 the whole page will change to "v41 to v4.2" update